specification/responses/4xx/messages/422_DuplicateMessageRequest.yaml

```
@@ -0,0 +1,21 @@
+description: |+
+  Request already received and it will be ignored.
+
+  ### Sandbox
+
+  It is possible to trigger this on the sandbox by using the `Prefer` header with a value of `code=422`.
+
+  Here is an example curl request to trigger a `422`:
+
+  ```
+    curl -X GET --header "Prefer: code=422_message" https://sandbox.api.service.nhs.uk/comms/
```

Does this need the full URL for the single message endpoint?
```suggestion
    curl -X GET --header "Prefer: code=422_message" https://sandbox.api.service.nhs.uk/comms/
```

---

Perhaps it _should_ but this is consistent with what's already there for 401, 403, 408, 425 etc, so probably ok for this PR imho.

---

Closing this as we're on an agreement unless I misunderstood.